File: AdministracionDeTienda/src/gestorAplicacion/clasesBase/Inventariar.java
package gestorAplicacion.clasesBase;

import java.util.ArrayList;

public interface Inventariar{

	int minCamisas=1;
	int minAbrigo=1;
	int minPantalon=1;

	
	public static int calcularCamisas(ArrayList<Producto> productos) {
		int numCamisas=0;
		for(Producto p:productos) {
			if(p.getNombre().equals("CAMISA") ){
				numCamisas++;
			}
		}
		return numCamisas;
	}
	
	public static int calcularAbrigos(ArrayList<Producto> productos) {
		int numAbrigos=0;
		for(Producto p:productos) {
			if(p.getNombre().equals("ABRIGO") ){
				numAbrigos++;
			}
		}
		return numAbrigos;
	}
	
	public static int calcularPantalon(ArrayList<Producto> productos) {
		int numPantalon=0;
		for(Producto p:productos) {
			if(p.getNombre().equals("PANTALON")) {
				
				numPantalon++;
				
			}
		}
		return numPantalon;
	}
	
	
}

File: Python/src/gestorAplicacion/clasesBase/Inventariar.py
from typing import List
from Producto import *

class Inventariar:
    minCamisas = 1
    minAbrigo = 1
    minPantalon = 1

    @staticmethod
    def calcularCamisas(productos) -> int:
        numCamisas = 0
        for p in productos:
            if p.getNombre() == Producto.Tipo.CAMISA:
                numCamisas += 1
        return numCamisas

    @staticmethod
    def calcularAbrigos(productos) -> int:
        numAbrigos = 0
        for p in productos:
            if p.getNombre() == Producto.Tipo.ABRIGO:
                numAbrigos += 1
        return numAbrigos

    @staticmethod
    def calcularPantalon(productos) -> int:
        numPantalon = 0
        for p in productos:
            if p.getNombre() == Producto.Tipo.PANTALON:
                numPantalon += 1
        return numPantalon